llpc/test/shaderdb/debug_info/DebugInfo_TestFsBasic.frag

```diff
@@ -33,11 +33,10 @@ void main()
 ; SHADERTEST: [[D7:![0-9]*]] = !{i32 2, !"Dwarf Version", i32 4}
 ; SHADERTEST: [[D8:![0-9]*]] = !{i32 2, !"Debug Info Version", i32 3}
 ; SHADERTEST: [[D9:![0-9]*]] = distinct !DICompileUnit(language: DW_LANG_C99, file: [[D10:![0-9]*]], producer: "spirv", isOptimized: false, runtimeVersion: 0, emissionKind: LineTablesOnly, enums: [[D11:![0-9]*]])
-; SHADERTEST: [[D10]] = !DIFile(filename: "spirv.dbg.cu", directory: ".")
+; SHADERTEST: [[D10]] = !DIFile(filename: "DebugInfo_TestFsBasic.frag", directory: "/vulkandriver/drivers/llpc/llpc/test/shaderdb/debug_info")
```

Comment:
How about using "{{.*}}" for the directory, because it will be different when people run the tests on their own machines?

Reply:
Thanks, will do.